devsetup/README.md

```diff
@@ -84,52 +84,7 @@ make download_tools
 ```
 
 ### EDPM deployment
-The EDPM deployment will create additional VM's alongside the crc VM, provides
-a mechanism to configure them using the ansibleee-operator.
-
-After completing the devsetup, attach the crc VM to the default network:
-```
-make crc_attach_default_interface
-```
-
-Deploy a compute node VM:
-```
-# Creates edpm-compute-0:
-make edpm_compute
-```
-
-Enable repositories:
-```
-make edpm_compute_repos
-```
-
-Execute the edpm_deploy step:
-```
-cd ..
-make edpm_deploy
-```
-
-You can also deploy additional compute node VMs:
-```
-# Set $EDPM_COMPUTE_SUFFIX to create additional VM's beyond 0:
-make edpm_compute EDPM_COMPUTE_SUFFIX=1
-```
-The IP of the compute node will be statically assigned starting at
-192.168.122.100 (based on the default EDPM_COMPUTE_SUFFIX=0).
-
-Then edit inventory in edpm/edpm-play.yaml.
-
-Cleanup:
-```
-make edpm_play_cleanup
-# Will delete VM's!:
-make edpm_compute_cleanup
-```
-
-In case additional compute node VMs are deployed, run:
-```
-make edpm_compute_cleanup EDPM_COMPUTE_SUFFIX=1
-```
+See [Deploy dev env using CRC, edpm nodes with isolated networks](../README.md#deploy-dev-env-using-crc-edpm-nodes-with-isolated-networks)
```

Comment:
`../README.md` doesn't say anything about having to run `make edpm_compute_repos` using the devsetup Makefile.

Reply:
because it is not needed anymore, as the repo-setup is service is being added

Reply:
I've removed this change, focus only on timeout for this PR